code/mob/living/critter/small_animal.dm

```diff
@@ -116,6 +116,7 @@ ABSTRACT_TYPE(/mob/living/critter/small_animal)
 		remove_lifeprocess(/datum/lifeprocess/sight)
 		remove_lifeprocess(/datum/lifeprocess/skin)
 		remove_lifeprocess(/datum/lifeprocess/statusupdate)
+		remove_lifeprocess(/datum/lifeprocess/radiation)
```

Comment:
Why

Reply:
No point ticking the lifeprocess after the critter is dead?